src/style.css

```diff
@@ -1 +1,113 @@
-/* add styles here */
+* {
+  margin: 0;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc) format('truetype-collection');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: regular;
```

Comment:
The `font-weight` property should use numeric values instead of 'regular'. Consider using '400' for normal weight.